_toc.yml

```diff
@@ -52,7 +52,12 @@ entries:
       - file: docs/tools/terraform/index
         title: Aiven Terraform provider
         entries:
-        - glob: docs/tools/terraform/*
+        - file: docs/tools/terraform/upgrade-provider-v1-v2
+        - file: docs/tools/terraform/reference/index
+          entries:
+          - file: docs/tools/terraform/reference/cookbook/index
+            entries:
+            - file: docs/tools/terraform/reference/cookbook/kafka-connect-terraform-recipe
```

Comment:
To make this easier to parse, I suggest that we add the title "Apache Kafka and OpenSearch" (no trademark symbols required in abbreviated navigation entries)

Reply:
NVM...the netlify preview renders the title correctly.